apps/communications/contacts/index.html

```diff
@@ -77,6 +77,7 @@
     <script defer src="/shared/elements/config.js"></script>
     <script defer src="/shared/js/component_utils.js"></script>
     <script defer src="/shared/elements/gaia_subheader/script.js"></script>
+    <script defer src="/shared/elements/gaia_switch/script.js"></script>
```

Comment:
Can we remove this script from the critical path and add it to the dependencies when we load the settings panel?